package.json

```diff
@@ -110,8 +110,6 @@
   },
   "jest": {
     "modulePathIgnorePatterns": [
-      "/.module-cache/",
-      "<rootDir>/build/",
```

Comment:
This was relevant when we had default root folder, and thus needed to exclude build explicitly. We don't anymore.